Realm/RLMObject.h

```diff
@@ -122,194 +127,201 @@ NS_ASSUME_NONNULL_BEGIN
+ Creates an instance of a Realm object with a given value, and adds it to the default Realm.
  
- If nested objects are included in the argument, `createInDefaultRealmWithValue:` will be called
+ If nested objects are included in the argument, `createInDefaultRealmWithValue:` will be recursively called
  on them.
 
- @param value   The value used to populate the object. This can be any key/value coding compliant
-                object, or a JSON object such as those returned from the methods in NSJSONSerialization, or
-                an NSArray with one object for each persisted property. An exception will be
-                thrown if any required properties are not present and no default is set.
+ @param value   A value used to populate the object. This can be any key/value coding compliant
+                object, or a JSON object such as those returned from the methods in `NSJSONSerialization`, or
+                an `NSArray` with one object for each persisted property. An exception will be
+                thrown if any required properties are not present and those properties were not defined with
+                default values.
 
- When passing in an NSArray, all properties must be present, valid and in the same order as the properties defined in the model.
+                When passing in an `NSArray`, all properties must be present, valid, and in the same order
+                as the properties defined in the model.
 
- @see   defaultPropertyValues
+ @see   `defaultPropertyValues`
  */
 + (instancetype)createInDefaultRealmWithValue:(id)value;
 
 /**
- Create an RLMObject in a Realm with a given object.
- 
- Creates an instance of this object and adds it to the given Realm populating
- the object with the given object.
+ Creates an instance of a Realm object with a given value, and adds it to the specified Realm.
  
- If nested objects are included in the argument, `createInRealm:withValue:` will be called
+ If nested objects are included in the argument, `createInRealm:withValue:` will be recursively called
  on them.
  
- @param realm   The Realm in which this object is persisted.
- @param value   The value used to populate the object. This can be any key/value coding compliant
-                object, or a JSON object such as those returned from the methods in NSJSONSerialization, or
-                an NSArray with one object for each persisted property. An exception will be
-                thrown if any required properties are not present and no default is set.
-                
-                When passing in an NSArray, all properties must be present, valid and in the same order as the properties defined in the model.
+ @param realm   The Realm which should manage the newly-created object.
+ @param value   A value used to populate the object. This can be any key/value coding compliant
+                object, or a JSON object such as those returned from the methods in `NSJSONSerialization`, or
```

Comment:
It feels strange to me to refer to "a JSON object" here. The objects returned from `NSJSONSerialization` are just plain ol' `NSDictionary`s and `NSArray`s. There's nothing JSONy about them.